docker/Dockerfile.capture

```diff
@@ -9,6 +9,19 @@ RUN apt-get update
 RUN apt-get -y upgrade
 
 
+# Intel VA-API drivers
+ARG TARGETARCH
```

Comment:
Not an arg to be passed in, part of `docker buildx` https://www.docker.com/blog/faster-multi-platform-builds-dockerfile-cross-compilation-guide/